src/openforms/forms/utils.py

```diff
@@ -320,7 +321,9 @@ def parse_trigger(trigger_info: JSONObject) -> Optional[JSONObject]:
         return trigger
 
 
-def parse_actions(actions: List[JSONObject]) -> List[JSONObject]:
+def parse_actions(
+    actions: List[JSONObject], component_key: str = None
```

Comment:
I left this with component_key=None so we don't need to update the detection script